### DISCUSSION TITLE
"Fooocus" for Larger Image Generation Without Distortion

### DISCUSSION THREAD

**Discussion by williamsblood** [lllyasviel/Fooocus#976] [Q&A]
Hello everyone,

I need some assistance with setting the "Fooocus" to generate larger images without causing distortion. Specifically, I'm trying to use a preset of 780x1280 with a scale of 1.5 for image generation. However, when I input 1152x1920, the images turn out distorted. Could anyone advise on the ideal pixel proportion for overriding this in developer mode? Any insights or tips would be greatly appreciated.

Thank you in advance!

**Comment by williamsblood** [comment]
@lllyasviel the original image didn't turn out with great quality despite using 60 steps. I experimented with 80 steps for 2x fast, 1.5 slow, and 2.0 slow, and noticed an improvement in the 'slow' mode. However, this improvement seems to be because the subject is well-known – specifically, the actor Ewan McGregor, as named in the prompt. When I use a prompt featuring an unknown individual and aim to enhance resolution to address imperfections in hands, eyes, etc., the photo often changes significantly. While upscaling mostly corrects these issues, the end result tends to differ substantially from the original. My question is: Could I generate this image in a larger size? My PC is capable of processing larger images – would this approach resolve the issue of imperfections? If not, could using a 1.5 upscale help to address this without altering the original result too much? Or do you have any other suggestions, such as an upcoming 'adetailer' or 'hi-res fix' feature, that might provide a solution?
![ewan slow 2x](https://github.com/lllyasviel/Fooocus/assets/144069983/144dedc5-9c0c-4065-85a9-b443ec4ae346)
![ewan slow 1 5x](https://github.com/lllyasviel/Fooocus/assets/144069983/2a642c71-df76-4e14-a9d4-0e97e63161b0)
![ewan fast 2x](https://github.com/lllyasviel/Fooocus/assets/144069983/09e035e9-b950-40f0-8c5a-7bbba361190b)
![orignal](https://github.com/lllyasviel/Fooocus/assets/144069983/7cb6d245-3f5b-4a19-9cf2-b3ff6ca84da5)

**Comment by williamsblood** [comment]
Another example, use this prompt: "Close-up of a young, white man as a rock climber, his face showing the exhilaration of scaling heights. His skin is sun-kissed, and his short hair is windswept, adding to his rugged, adventurous look. The vast expanse of the cliff and the sky behind him create a dynamic backdrop that highlights his determination and the thrill of climbing." 80 steps here
![Screenshot 2023-11-19 134601](https://github.com/lllyasviel/Fooocus/assets/144069983/e62ddc76-6370-47f6-a077-59120ef1ca6e)

**Reply by williamsblood** [reply]
original generated
![escalador original](https://github.com/lllyasviel/Fooocus/assets/144069983/6b6d015c-7057-4853-a415-7d662de07516)
 here

**Reply by williamsblood** [reply]
fast here

![fast2x](https://github.com/lllyasviel/Fooocus/assets/144069983/e59d7ca1-bdbf-43a2-a672-b0af707c249d)

**Reply by williamsblood** [reply]
1.5 slow 
![1 5 slow](https://github.com/lllyasviel/Fooocus/assets/144069983/3e6cd57d-fa69-4186-8b67-afa22701b873)

**Reply by williamsblood** [reply]
Sometimes I can't fix the eyes even with slow upscaling.

**Comment by lllyasviel** [comment]
see also https://github.com/lllyasviel/Fooocus/discussions/414#detail

**Reply by williamsblood** [reply]
Ty! I will test soon!
